ALUM/ALUM/Services/ServiceHelper.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  RequestGenerator.swift
+//  ALUM
+//
+//  Created by Yash Ravipati on 4/24/23.
+//
+
+import Foundation
+
+class ServiceHelper {
+    static let shared = ServiceHelper()
+
+    func attachAuthTokenToRequest(request: inout URLRequest) async throws {
```

Comment:
Since a lot of our requests will require the firebase auth to be attached, this function will handle that so that we don't have to figure out how to get the token and attach to request everytime.